antarest/study/storage/abstract_storage_service.py

```diff
@@ -239,11 +244,11 @@ def import_output(
         path_output = (
             Path(metadata.path) / "output" / f"imported_output_{str(uuid4())}"
         )
-        os.makedirs(path_output)
+        path_output.mkdir(parents=True)
         output_full_name: Optional[str] = None
         try:
             if isinstance(output, Path):
-                if output != path_output:
+                if output != path_output and output.suffix != ".zip":
```

Comment:
So there if the output is a zip it is never imported ?
It should be copied in the path_output and renamed using the extract_output_name method